client/src/main/java/io/pravega/client/admin/impl/ReaderGroupManagerImpl.java

```diff
@@ -98,18 +97,16 @@ public boolean createReaderGroup(String groupName, ReaderGroupConfig config) thr
         ReaderGroupConfig controllerConfig = getThrowingException(controller.createReaderGroup(scope, groupName, config));
         if (!controllerConfig.equals(config)) {
             log.warn("ReaderGroup {} already exists with pre-existing configuration {}", groupName, controllerConfig);
-            throw new ConfigMismatchException(groupName, controllerConfig);
```

Comment:
with this change we are returning a new configuration.
but this is dangerous because it is very hard to understand from the returned value if the request configuration has been applied or not.
the user must compare the configuration (returned vs given) in order to understand if the configuration is the expected one


IIUC this is not much user friendly and error prone